src/emulator/download.js

```diff
@@ -13,6 +14,11 @@ module.exports = (name) => {
     let req = request.get(emulator.remoteUrl);
     let writeStream = fs.createWriteStream(emulator.localPath);
     req.on("error", (err) => reject(err));
+    req.on('response', response => {
```

Comment:
```suggestion
    req.on('response', (response) => {
```

should fix the formatter.